src/components/StorePage/styles/Checkout.css

```diff
@@ -0,0 +1,92 @@
+.checkoutImage {
+  width: 100%;
+}
+
+.checkoutDescription {
+  font-family: 'ArialRoundedMTBold';
```

Comment:
Is it possible to apply `font-family: 'ArialRoundedMTBold';` to a parent element so that the default is to use that font? Then, we can refrain from definining the font however many times you need to, and just keep it in one place. Would also make updating the font much easier. You could probably do this for `font-size`, too. Apply the size to some all-encompassing parent element, and then only define `font-size` again for elements that neef sizes other than `22px`.